api/src/content/content.types.ts

```diff
@@ -0,0 +1,185 @@
+import { Block, Inline } from '@contentful/rich-text-types'
+import { EntrySys, FieldsType } from 'contentful'
+import { Content } from '@prisma/client'
+
+// Pulled from https://github.com/lbenie/reading-time-estimator/blob/a5c57383575d8dda37f4d501c57f9e7658db6f0a/lib/reading-time-estimator.ts#L7-L20
+//  Since the author didn't seem to think exporting the type was necessary, I had to copy it here 😡
+type ReadingTime = {
+  readonly minutes: number
+  readonly words: number
+  readonly text: string
+}
+
```

Comment:
the new blog design is not using "reading time" you can remove this

Reply:
![stone-cold-oh-hell-yeah](https://github.com/user-attachments/assets/b600109d-4b81-4f88-8dfc-c85c3ede45b5)